src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1037,6 +1037,8 @@ CurrentToken.Kind is (SyntaxKind.GreaterThanToken or SyntaxKind.GreaterThanEqual
                                 operatorToken.Text + operatorToken2.Text + operatorToken3.Text,
                                 operatorToken.ValueText + operatorToken2.ValueText + operatorToken3.ValueText,
                                 operatorToken3.GetTrailingTrivia());
+
+                            operatorToken = CheckFeatureAvailability(operatorToken, MessageID.IDS_FeatureUnsignedRightShift, forceWarning: true);
```

Comment:
nit: Why do a parser LangVer check as opposed to add check in Binder_Cref?

Reply:
> Why do a parser LangVer check as opposed to add check in Binder_Cref?

Just following the established pattern.

Reply:
Note that these are warnings, not errors